perfkitbenchmarker/disk.py

```diff
@@ -21,6 +21,11 @@
 
 from perfkitbenchmarker import resource
 
+STANDARD = 'standard'
+SSD = 'ssd'
+IOPS = 'iops'  # Provisioned IOPS (ssd) in AWS
+LOCAL = 'local'
```

Comment:
From PKB's point of view, this is very hard to tell the difference unless we start keeping track of which machine types have which kind of local drive (or check at run time whether the device says it's rotational or not). The intention was that local means whatever local drive you happen to have, ssd or rotational. Did you have an idea about how you wanted it to behave if someone specified --scratch_disk_type=local_ssd and the vm only had a rotational disk?